Makefile

```diff
@@ -36,26 +36,26 @@ profile: FORCE
 	bash scripts/profile_model.sh ${ref} ${models}
 
 test: lint docs doctest FORCE
-	pytest -vx -n auto --stage unit
+	pytest -v -n auto --stage unit
 
 test-examples: lint FORCE
-	pytest -vx -n auto --stage test_examples
+	pytest -v -n auto --stage test_examples
 
 test-tutorials: lint FORCE
 	grep -l smoke_test tutorial/source/*.ipynb | xargs grep -L 'smoke_test = False' \
-		| CI=1 xargs pytest -vx --nbval-lax --current-env
+		| CI=1 xargs pytest -v --nbval-lax --current-env
```

Comment:
As @fritzo mentioned, all of these test stages are quite expensive with a fair amount of generated log, which motivates the current default to stop at the first failure. We have documented some of the ways to run tests in our [contributing](https://github.com/pyro-ppl/pyro/blob/dev/CONTRIBUTING.md) guide. Please feel free to add to that instead.